helper/validation/web.go

```diff
@@ -0,0 +1,52 @@
+package validation
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func URLIsHTTPS(i interface{}, k string) (_ []string, errors []error) {
```

Comment:
I like the way this flows... but it feels inconsistent, anyway this can be reworded to being with `Is`... maybe
```
IsHTTPSURL
```
and
```
IsHTTPURL
```